scripts/buildah-bud-push.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+shopt -s inherit_errexit
+set -eu -o pipefail
+
+source "$(dirname ${BASH_SOURCE[0]})/buildah-prepare.sh"
+
+phase "Executing buildah bud script"
+
+set -x
```

Comment:
Nit: not sure we want this by default (maybe a verbose param ?), as.. people could pass some sort of secret through `PARAMS_BUILD_EXTRA_ARGS` that would leak into the logs.

Reply:
Yeah sound fair to me, so what I will do is remove the `set -x` , and add a verbose param, would be useful for logging any tests.